package.json

```diff
@@ -51,6 +51,7 @@
     "prettier-plugin-solidity": "^1.0.0-beta.1",
```

Comment:
Seems like we should also bump `prettier-plugin-solidity` to 1.0.0-beta.7 which introduces a fix for breaking solidity 8 changes: https://github.com/prettier-solidity/prettier-plugin-solidity/releases/tag/v1.0.0-beta.7

(pull request #459)

Reply:
yup! as discussed offline though we should do this as a separate PR.